kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/B2BPurchaseOrderSciquestServiceImpl.java

```diff
@@ -0,0 +1,375 @@
+        String phoneNumber = purchaseOrder.getBillingPhoneNumber();
+        String billingLine1 = purchaseOrder.getBillingLine1Address();
+        String billingLine2 = purchaseOrder.getBillingLine2Address();
+        String city = purchaseOrder.getBillingCityName();
+        String stateCode = purchaseOrder.getBillingStateCode();
+        String postalCode = purchaseOrder.getBillingPostalCode();
+        String countryCode = purchaseOrder.getBillingCountryName();
+        String countryIsoCode = purchaseOrder.getBillingCountryCode();
+
+        /** *** BILL TO SECTION **** */
+        // Contact - There can be 0-5 Contact elements. The label attribute is optional.
+        // There must be 1-5 AddressLine elements. The label attribute is optional.
+        cXml.append("      <BillTo>\n");
+        cXml.append("        <Address>\n");
+        cXml.append("          <TemplateName>Bill To</TemplateName>\n");
+        // cXml.append("          <AddressCode>"+bn(addressCode)+"</AddressCode>\n");
+        cXml.append("          <Contact label=\"Phone\" linenumber=\"1\"><![CDATA[Phone: " + bn(phoneNumber) + "]]></Contact>\n");
+        cXml.append("          <Contact label=\"Fax\" linenumber=\"2\"><![CDATA[Fax: 520-621-1245]]></Contact>\n");
+        cXml.append("          <Contact label=\"Company\" linenumber=\"3\"><![CDATA[" + bn(companyName) + "]]></Contact>\n");
+        cXml.append("          <AddressLine label=\"Street1\" linenumber=\"1\"><![CDATA[University Services Building]]></AddressLine>\n");
+        cXml.append("          <AddressLine label=\"Street2\" linenumber=\"2\"><![CDATA[" + bn(billingLine1) + "]]></AddressLine>\n");
+        cXml.append("          <AddressLine label=\"Street3\" linenumber=\"3\"><![CDATA[Building 0158]]></AddressLine>\n");
+        cXml.append("          <AddressLine label=\"Street4\" linenumber=\"4\"><![CDATA[" + bn(billingLine2) + "]]></AddressLine>\n");
+        cXml.append("          <City><![CDATA[" + bn(city) + "]]></City>\n"); // Required.
+        cXml.append("          <State>" + bn(stateCode) + "</State>\n");
+        cXml.append("          <PostalCode>" + bn(postalCode) + "</PostalCode>\n"); // Required.
+        cXml.append("          <Country isocountrycode=\"" + bn(countryIsoCode) + "\">" + bn(countryCode) + "</Country>\n");
+        cXml.append("        </Address>\n");
+        cXml.append("      </BillTo>\n");
+
+        return cXml;
+    }
+
+    protected StringBuilder writeShipToSection(StringBuilder cXml, PurchaseOrderDocument purchaseOrder, Person initiator) {
+
+        // AddressCode. A code to identify the address, that is sent to the supplier.
+        String addressCode = purchaseOrder.getDeliveryCampusCode() + purchaseOrder.getOrganizationCode();
+        String contactLine1 = StringUtils.isNotBlank(purchaseOrder.getDeliveryBuildingRoomNumber()) ? "Rm " + purchaseOrder.getDeliveryBuildingRoomNumber() : "";
+        // START KFSI-5802

```

Comment:
Same as the KFSI-750 comment above.  Do we know if this was fixed in KFS6 baseline to know if this is actually needed?